index.html

```diff
@@ -0,0 +1,117 @@
+          <a id="hamburger"><img src="img/mobile_menu.png" alt=""></a>
+          <div id="mobile-menu-list" class="mobile-menu-list d-none col-12 pt-1">
+            <a class="close-btn text-white" href=""> &times;</a>
+            <a class="orange-text" href="about.html">About</a>
+            <a class="gray-text" href="">Program</a>
+            <a class="gray-text" href="">Join</a>
+            <a class="gray-text" href="">Sponsor</a>
+            <a class="gray-text" href="">News</a>
+            <a class="orange-text orange-border" href="">CC Campaign</a>
+          </div>
+        </div>
+        <div class="pt-1 logo">
+          <a class="pr-5" href="index.html"><img src="img/logo.png"></a>
+        </div>
+        <div class="d-none d-md-flex mt-2 col-10 justify-content-around pt-1">
+          <a class="pt-3 orange-text" href="about.html">About</a>
+          <a class="pt-3 gray-text" href="">Program</a>
+          <a class="pt-3 gray-text" href="">Join</a>
+          <a class="pt-3 gray-text" href="">Sponsor</a>
+          <a class="pt-3 gray-text" href="">News</a>
+          <a class="pt-2 pr-3 pb-2 pl-3 orange-text orange-border" href="">CC Campaign</a>
+        </div>
+      </section>
+    </nav>
+  </header>
+  <main id="main">
+    <section class="landing-page">
+      <section class="title col-12 col-md-6 mb-md-5">
+        <h2 class="orange-text small-title">"Hello! Tech World"</h2>
+        <h1 class="orange-text big-title">GOOGLE DEVELOPERS CONTINENTAL SUMMIT 2023</h1>
+      </section>
+      <section class="sub-header col-11 col-md-6 ml-2 ml-md-3 pt-md-3 pb-md-3 mb-md-5 bg-light-grey">
+        <p>A joyful celebration believing in the value of openness and sharing, creating a 
+          positive change with people from over 80 countries is taking place in October, in Korea.
+        </p>
+      </section>
+      <section class="date-location col-11 col-md-5 ml-3 pt-4 pt-md-2 pt-md-3 pb-3 mb-2">
+        <p class="date">2023.03.28(TUE) ~ 31(FRI)</p>
+        <p class="location">@ Science Museum of Ethiopia, Ethio Telecom Hall</p>
+      </section>
+    </section>
+    <section class="main-program mb-5 d-flex flex-column align-items-center justify-content-center text-center">
+      <h3 class="text-white main-program-title">Main Program </h3>
+      <section class="card-container d-flex flex-column flex-md-row justify-content-around text-center align-items-center">
+        <div class="d-flex flex-row flex-md-column card-1 bg-card align-items-center justify-content-center p-1">
+          <img class="card-logo" src="img/program_icon_01.png">
+          <h4 class="card-title orange-text">Lecture</h4>
+          <p class="card-description">Listen to the speakers from various countries about the messages of sharing and openign.</p>
+        </div>
+        <div class="d-flex flex-row flex-md-column card-2 bg-card align-items-center justify-content-center p-1">
+          <img class="card-logo" src="img/program_icon_02.png">
```

Comment:
- [ ] Hi, all images in both Home and About pages must contain `alt` attributes. I would advise you to fix that.
- [ ] NB: What does alt do in HTML IMG?
The required alt attribute specifies an alternate text for an image, if the image cannot be displayed. The alt attribute provides alternative information for an image if a user for some reason cannot view it (because of slow connection, an error in the src attribute, or if the user uses a screen reader).